databases/week2/Part 2 School database .SQL

```diff
@@ -0,0 +1,63 @@
+-- Create a new database containing the following tables:
+Class: with the columns: id, name, begins (date), ends (date)
+Student: with the columns: id, name, email, phone, class_id (foreign key)
+
+SET NAMES utf8mb4;
+
+
+
+
+CREATE TABLE `class1` (
```

Comment:
It's not good practice to have numbers in your table names. The same for the `student` table